cmd/otelcol/config/collector/config.d.linux/receivers/mysql.discovery.yaml

```diff
@@ -18,26 +18,26 @@
 #       password: splunk.discovery.default
 #   status:
```

Comment:
This can be changed to something like `status_detection` as the next step to avoid different `status` fields

Reply:
Is this change absolutely necessary for entity events?

Reply:
It's unnecessary, but having two status fields is confusing. The higher level `status` doesn't seem like a good name for what it's defining. Do you have any concerns with renaming it?

Reply:
Giving that we are significantly changing this interface, I don't see why we need to restrain any further changes

Reply:
Understood!

Reply:
I think I would go with your change as it makes things more deterministic.